### ISSUE TITLE
Consider adding some Quality-of-Life features like auto-switch-to-720p and auto-enable-captions

### ISSUE THREAD

**Issue by sagz** [Fugiman/google-meet-grid-view#118]
Would love the extension to improve my overall Meet experience with having options to turn on captions automatically (Currently, I have to enable manually in every meeting, even if back to back and on the same browser)

Also, the new consumer Meet that's recently launched defaults to 360p for send and receive and, again, I have to manually switch it to 720p each and every time. (Since I have the required compute and bandwidth).

I understand it might be out of scope of the 'grid view' featureset of this extension, but would be great addition to have these QoL features.
They are relatively low-dev-effort and high-user-benefit.

**Comment by MarcAnt01** [comment]
I totally agree, I’d love to have an option for high res enforced, instead if having to change it every time

**Comment by sagz** [comment]
Ping! any comment @Fugiman

**Comment by Fugiman** [comment]
I briefly looked into this and couldn't find an easy way to do it. It certainly isn't "low dev effort". I'll probably look into it again eventually but it is very low priority for me.

**Comment by MarcAnt01** [comment]
@sagz what?

**Comment by sagz** [comment]
Thanks for checking @Fugiman . I stand corrected about it being low-dev-effort. My bad there.
